src/Zicht/Bundle/AdminBundle/Resources/views/sonata_layout.html.twig

```diff
@@ -1,2 +1,43 @@
 {% extends 'SonataAdminBundle::standard_layout.html.twig' %}
 
+{% block stylesheets %}
+    {{ parent() }}
+
+    <link rel="stylesheet" href="{{ asset('bundles/zichtadmin/style/admin-cms.css') }}">
+
+    <style>
+
+        .logo svg {
+            display: inline;
+            padding-bottom: 4px;
+            max-height: 100%;
+            max-width: 60px;
+        }
+
+        .logo span {
+            display: inline-block;
+            line-height: 1;
+            vertical-align: middle;
+            width: 200px;
+        }
+
+        .logo span.cms_title {
```

Comment:
Misschien dat deze lege declaratie weg mag, of is dat een placeholder voor mogelijke styling op projectbasis?